.vsts-ci.yml

```diff
@@ -100,15 +100,11 @@ steps:
 
 # Publish VSTS artifact that the RoslynInsertionTool is able to find by its name.
 # The backing storage of the artifact is a file share.
-- task: PublishBuildArtifacts@1
+- task: PowerShell@2
```

Comment:
This won't work. `PublishBuildArtifacts` not only copies files but also associates ArtifactName with the share path in VSTS artifact database. The insertion tool depends on the artifacts being available in the database.

Reply:
We can change the step below (`Publish Drop - VSSetup`) to PublicBuildArtifacts.

Reply:
@tmat can you elaborate a bit? I'm unsure what change I need to make here.

Reply:
Actually, it won't be that simple. We need to first update Roslyn insertion tool. 
Then we need to make some changes to the `Publish Drop - VSSetup` task below.

@jmarolf